src/controllers/midi/portmidienumerator.cpp

```diff
@@ -201,7 +201,7 @@ QList<Controller*> PortMidiEnumerator::queryDevices() {
         qDebug() << " Found output device"
                  << "#" << i << pDeviceInfo->name;
         QString deviceName = pDeviceInfo->name;
-        unassignedOutputDevices[i] = deviceName;
+        unassignedOutputDevices[i] = std::move(deviceName);
```

Comment:
Is this a false positive? Operator[] takes a const &. 
Maybe call insert() as a workaround?